pkg/kubelet/kubelet_pods.go

```diff
@@ -245,10 +245,10 @@ func (kl *Kubelet) makeBlockVolumes(pod *v1.Pod, container *v1.Container, podVol
 // - container is not an infrastructure (pause) container
 // - container is not already mounting on /etc/hosts
 // Kubernetes will not mount /etc/hosts if:
-// - when the Pod sandbox is being created, its IP is still unknown. Hence, PodIP will not have been set.
-// - Windows pod contains a hostProcess container
+// - the Pod is on the pod network and PodIP has not yet been set.
```

Comment:
Do you think keeping the description like "`(e.g., Pod sandbox is being created)`" here may help people understand what it means?

Reply:
happy to take any suggestions, it is clear that depends a lot from the area of the reader, see
https://github.com/kubernetes/kubernetes/pull/126460#discussion_r1697011449

IF needed, we just duplicate the comment with both wordings

Reply:
> IF needed, we just duplicate the comment with both wordings

Sounds good. It's just a minor nit to preserve the original information. Thanks.

Reply:
updated @yujuhong